2-mandatory/2-financial-times.js

```diff
@@ -23,15 +50,45 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    const newarray=[];
+    const number=/[0-9]/;
+let i=0;
+ while (i < allArticleTitles.length) {
+   if (allArticleTitles[i].match(number)) {
+     newarray.push(allArticleTitles[i]);
+   }
+   i=i+1;
+ }
+return newarray;
 }
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 /*
     The Financial Times wants to understand what the average number of characters in an article title is.
     Implement the function below to return this number - rounded to the nearest integer.
 */
 function averageNumberOfCharacters(allArticleTitles) {
-    // TODO
+   let sum=0;
+    for (const item of allArticleTitles) {
+     sum = (item.length)+sum;
```

Comment:
why is item.length to be inside the () ? 
is there another way to do a 'sum = variable + sum'? a shorter way?

Reply:
sum += item.length